public/scss/_fonts.scss

```diff
@@ -0,0 +1,59 @@
+/* lato-italic - latin */
+@font-face {
```

Comment:
I actually was going to provide feedback on all these font-face blocks.
I recommend adding the `font-display` property and setting it to `swap`.

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display

Reply:
Essentially, this allows for rendering the webpage earlier (before fonts are retrieved). So, it makes the website look more responsive.

Reply:
@tomudding

Reply:
I don't see why this would suddenly be a problem. When we used the Google CDNs this was never a problem and the fonts are still cached similarly (although we might want to change the NGINX config to actually enforce it).

The biggest blocking resource is the `gewis-theme.css`, as long as that is not loaded there won't be any fonts anyway.

Reply:
But please do make a PR and I'll merge it.

Reply:
@Koen1999 Perhaps we should preload the fonts too:

```html
<head>
    <!-- ... -->
    <link rel="preload" href="/raleway/raleway-v22-latin-500.woff2" as="font" type="font/woff2">
    <!-- ... -->
</head>
```

Reply:
To really prioritise loading them as early as possible.